app/config/security_test.yml

```
@@ -13,7 +13,7 @@ security:
     oauth2:
       memory:
         users:
-          my_client_id: { password: 'prestashop' }
+          my_client_id: { password: 'prestashop', roles: [ 'ROLE_HOOK:READ' ] }
```

Why is the role written this way capitalized? Is it because there's a system somewhere that automatically uppercases the scope value? (and appends `ROLE_` in front of it?)

I was expecting we could do:

```suggestion
          my_client_id: { password: 'prestashop', roles: [ 'Hook:read' ] }
```

with the name of the scope directly used as the role name. But from what you show here it seems Symfony roles have some mandatory format always in capitalized characters? But which component converts this automatically then?